test/acceptance/stubbing_non_existent_any_instance_method_test.rb

```diff
@@ -67,7 +67,7 @@ def test_should_allow_stubbing_method_to_which_any_instance_responds
     Mocha::Configuration.prevent(:stubbing_non_existent_method)
     klass = Class.new do
       def respond_to?(method, _include_private = false)
-        (method == :method_to_which_instance_responds)
+        (method.to_sym == :method_to_which_instance_responds)
```

Comment:
I'm not 100% sure I understand why these tests (and the other ones in this commit) have to change. Can you explain a bit more?

In any case I'd prefer if all the changes to the tests in this commit were incorporated into the commit which actually required them.

Reply:
Resolving - the whole branch has been re-written.